db/schema.rb

```diff
@@ -1439,9 +1440,9 @@
 
   create_table "users_gdpr_delete_requests", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
-    t.string "email"
+    t.string "email", null: false
```

Comment:
I did not make this change

Reply:
Then please remove `db/schema.rb` from this PR.